src/Data/Functor/Contravariant/Divisible.hs

```diff
@@ -603,3 +607,58 @@ instance Decidable SettableStateVar where
 -- In addition, we expect the same kind of distributive law as is satisfied by the usual
 -- covariant 'Alternative', w.r.t 'Applicative', which should be fully formulated and
 -- added here at some point!
+
+-- | Analagous to `(*>)`
+--
+-- @
+-- showing :: 'Show' a => 'Op' 'String' a
+-- showing = 'Op' 'show'
+--
+-- string :: String -> 'Op' 'String' ()
+-- string = 'Op' '.' 'const'
+--
+-- greeting :: Show a => 'Op' 'String' a
+-- greeting = string "Hello " '>*' showing
+-- @
+(>*) :: Divisible f => f () -> f a -> f a
+before >* p = ((),) >$< before >*< p
+
+infixr 5 >*
+
+-- | Analagous to `(<*)`
+--
+-- @
+-- emphatic :: 'Op' 'String' a -> 'Op' 'String' a
+-- emphatic opstring = opstring '*<' string "!"
+-- @
+(*<) :: Divisible f => f a -> f () -> f a
+p *< after = (,()) >$< p >*< after
+
+infixr 5 *<
```

Comment:
The @gwils talk uses `infixr 4`. Not saying that one is correct, but is there a reason?

https://youtu.be/IJ_bVVsQhvc?t=1311

Reply:
I haven't seen this talk yet. My reasoning was by analogy. The `Applicative` operators are all `infixl 4`.
```Haskell
infixl 4 <*>
infixl 4 <*
infixl 4 *>
```
The `Contravariant` operator so far is `infixr 5 >*<` so I gave `>*` and `*<` the same fixity. I'm not sure that's "correct".

Reply:
Looks like I switched the signatures of `<*` and `*>` from that talk too, oops.